Laminin enterocyte of colon

JABelfiore opened the pull request.

dosumis commented:
#gogoeditdiff

Caroline-99 commented:
This PR is still in progress - Need new references to support the definition

Caroline-99 commented on src/ontology/cl-edit.owl:
```diff
@@ -32547,6 +32548,15 @@ AnnotationAssertion(terms:date obo:CL_4047019 "2024-09-24T12:11:09Z"^^xsd:dateTi
 
+# Class: obo:CL_4047021 (laminin enterocyte of colon)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:14585148") Annotation(oboInOwl:hasDbXref "PMID:35101094") obo:IAO_0000115 obo:CL_4047021 "An enterocyte located in the epithelium of the large intestine. This cell expresses and interacts with specific laminin isoforms, particularly laminin-332 (composed of α3, β3, and γ2 chains) and laminin-511 (composed of α5, β1, and γ1 chains). Laminin e ...
```
These references are not relevant - new references are needed to support the definition.

dosumis commented:
Please fix PMIDs first as this will help discussion.

JABelfiore commented:
Updated name and subclass to suggested. Replaced PMID with PMID on laminin isoforms. however still needs another reference mentioning laminin colonocytes or laminin enterocyte of colon by name. I couldn't find these in literature, spoke to GCA and they also don't have current papers on this. **Action:** Icebox ticket and create list of new cells not in literature for GCA to create a publication on
